src/shipchain_common/test_utils.py

```diff
@@ -46,9 +46,9 @@ def datetimeAlmostEqual(dt1, dt2=None, ms_threshold=None):
     return dt1 - timedelta(milliseconds=ms_threshold) <= dt2 <= dt1 + timedelta(milliseconds=ms_threshold)
 
 
-def get_jwt(exp=None, sub='00000000-0000-0000-0000-000000000000', username='fake@shipchain.io', **kwargs):
+def get_jwt(exp=None, iat=None, sub='00000000-0000-0000-0000-000000000000', username='fake@shipchain.io', **kwargs):
     payload = {'email': username, 'username': username, 'sub': sub,
-               'aud': '892633'}
+               'aud': '892633', 'jti': uuid4().hex}
```

Comment:
I forget - should we add `jti` to our tokens generated by Profiles to make caching a little cleaner? I assume it'll use the sub.exp approach for our existing tokens.

Reply:
In my Profiles branch which utilizes this functionality, `jti` is included in every generated token using the `openid` scope.  As this a utility function for unit tests, the tokens are not actually generated by Profiles here and the `jti` is being added explicitly.

Reply:
:thinking: I'm not sure why I don't see `jti` in my tokens. Sounds good though.